CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ endif()
 if(APPLE)
   if(TC_BUILD_IOS)
     SET(TC_BASE_SDK iphoneos)
-    SET(COMPILER_FLAGS "${COMPILER_FLAGS} -miphoneos-version-min=12.0 -Wno-c++11-narrowing")
+    SET(COMPILER_FLAGS "${COMPILER_FLAGS} -miphoneos-version-min=11.0 -Wno-c++11-narrowing")
```

Comment:
Thinking about this some more, does everything still work if we set this to 10.0?

Then it better matches the macOS deployment target (will be the iOS that was released the same year as that macOS).

It may also help ensure that all CoreML symbols are weak-linked, making it more likely that our dylibs can be included in apps targeted older OS versions (even ones that don't include CoreML, in case all the CoreML features are optional).